### ISSUE TITLE
lua-mpack 1.0.2 version doesn't exist and I am confused.

### ISSUE THREAD

**Issue by rtlanceroad** [6c37/crux-ports#61]
If anyone know what 1.0.2 means, let me know. thanks.

**Comment by onodera-punpun** [comment]
https://www.archlinux.org/packages/community/x86_64/lua51-mpack/

https://github.com/tarruda/libmpack

**Comment by rtlanceroad** [comment]
lua-mpack has 5.1 , 5.2, and 5.3 compatibility, I think commit 921b200033138735f3be3029ef63faca9550912d has implemented this.

For libmpack which i think 1.0.2 means this, I think we need a seperate pkg.

**Comment by rtlanceroad** [comment]
I dont know, maybe commit 25aa2abe5d2cc2c20db62710be0b10b555c57596 is kinda fix?

**Comment by onodera-punpun** [comment]
I think so, why do we need both though? What package depends on libmpack and what on lua-mpack?

**Comment by rtlanceroad** [comment]
IMOP, I just bump them as an maintainer. : -)
You are right. they are both implementions for msgpack, the only difference is one is in Lua format. the other is in library format. We _really_ dont need both, and no package depends on them since they are just an serialization library for msgpack. Feel free to pick one or delete both.

**Comment by maraku** [comment]
hey there.
for me, neovim fails to build with "no lua compiler" (or something like that) when libmpack isn't installed and lua-mpack is installed. When I installed libmpack, nvim builds fine. furthermore, I removed lua-mpack but kept libmpack, and neovim still could build.
TL;DR libmpack, not lua-mpack (at least for neovim)
do tell if I'm doing something wrong here...

**Comment by rtlanceroad** [comment]
@maraku Thanks for your feedback, you are with @onodera-punpun :-)
I think no package depends on lua-mpack and lua-mpack depends on nothing. but at least neovim depends on libmpack. so I'd like to invite onodera to do following:
1. purge lua-mpack from our repo
or 
2. throw it to -orphan repo

I am just a monkey bumping our pkgs :)

**Comment by onodera-punpun** [comment]
Done :)

**Comment by maraku** [comment]
could you also replace lua-mpack with libmpack in the dependencies for neovim?
thanks, @onodera-punpun and @rtlanceroad

**Comment by rtlanceroad** [comment]
@maraku Thanks for noticing, the commit 4efa224456632d5efaaf1f7f2f43c21fb1ee3f81 should fix it.
